source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -210,6 +214,15 @@ void Filter::onComplete(Filters::Common::ExtAuthz::ResponsePtr&& response) {
   Stats::StatName empty_stat_name;
 
   if (!response->dynamic_metadata.fields().empty()) {
+    // Add duration of call to dynamic metadata if applicable
+    StreamInfo::TimingUtility timingUtility(decoder_callbacks_->streamInfo());
+    auto ext_authz_duration = timingUtility.recordDuration(start_time_);
+    if (ext_authz_duration.has_value()) {
+      ProtobufWkt::Value ext_authz_duration_value;
+      ext_authz_duration_value.set_number_value(ext_authz_duration->count());
+      (*response->dynamic_metadata.mutable_fields())["duration"] = ext_authz_duration_value;
```

Comment:
Since this "duration" field is set by the filter (not by the ext_authz service), could you help to document it in https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter#dynamic-metadata? Thanks!

Reply:
Added documentation.